packages/frontend/src/config/environmentDefaults/mainnet.js

```diff
@@ -19,6 +19,9 @@ export default {
     INDEXER_SERVICE_URL: 'https://api.kitwallet.app',
     LINKDROP_GAS: '100000000000000',
     LOCKUP_ACCOUNT_ID_SUFFIX: 'lockup.near',
+    MIGRATION_CALENDAR_EVENT: 'https://calendar.google.com/event?action=TEMPLATE&tmeid=MGkwbjMxcDI5bWJxaHEzazQwdGs0a244YXIgcGhpbGlwLmMub2Jvc2lAbQ&tmsrc=philip.c.obosi%40gmail.com',
```

Comment:
This does not appear to be used anywhere and is set in all the `environmentDefaults` files

Reply:
Yeah. Switched the implementation.